dotCMS/src/main/java/com/dotmarketing/util/InodeUtils.java

```diff
@@ -171,4 +178,34 @@ public static String getAssetTypeFromDB(String inode) throws DotDataException {
 
 		return assetType;
 	}
+	
+	/**
+	 * This method intents to replace any call to the deprecated InodeFactory.getInode,
+	 * since the Structure mapping was deleted from the Hibernate files, now you need to 
+	 * get it via the ContentTypeAPI.
+	 * 
+	 * @param inode
+	 * @return
+	 * @throws DotDataException
+	 * @throws DotSecurityException
+	 */
+	public static Inode getInode(String inode) throws DotDataException, DotSecurityException{
+	    Inode inodeObj = null;
+        //Using the ShortyAPI to identify the nature of this inode
+        Optional<ShortyId> shortOpt = APILocator.getShortyAPI().getShorty(inode);
+        
+      //Hibernate won't handle structures, thats why we need a special case here
+        if ( shortOpt.isPresent() && ShortType.STRUCTURE == shortOpt.get().subType ) {
+
+            //Search for the given ContentType inode
+            ContentType foundContentType = APILocator.getContentTypeAPI(APILocator.systemUser()).find(inode);
+            if ( null != foundContentType ) {
+                //Transform the found content type to a Structure
+                inodeObj = new StructureTransformer(foundContentType).asStructure();
+            }
+        } else {
+            inodeObj = InodeFactory.getInode(inode, Inode.class);

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "InodeFactory"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)